ui/src/app/pages/admin/admin-institution-edit.spec.tsx

```diff
@@ -149,13 +153,96 @@ describe('AdminInstitutionEditSpec - edit mode', () => {
     expect(wrapper).toBeTruthy();
 
     expect(findCTDetails(wrapper).exists()).toBeTruthy();
+    expect(textInputValue(findCTAddressInput(wrapper))).toBe('foo@verily.com');
 
     await simulateComponentChange(wrapper, findCTEnabled(wrapper), false);
+    expect(findCTEnabled(wrapper).props.checked).toBeFalsy();
     expect(findCTDetails(wrapper).exists()).toBeFalsy();
+    expect(findCTAddressInput(wrapper).exists()).toBeFalsy();
 
     await simulateComponentChange(wrapper, findCTEnabled(wrapper), true);
     expect(findCTEnabled(wrapper).props.checked).toBeTruthy();
     expect(findCTDetails(wrapper).exists()).toBeTruthy();
+    expect(textInputValue(findCTAddressInput(wrapper))).toBe('foo@verily.com');
```

Comment:
updated test to catch the issue you saw @aweng98

Reply:
👍🏻